src/main/java/com/mariamkatamashvlii/gym/profile/Dev.java

```diff
@@ -0,0 +1,9 @@
+package com.mariamkatamashvlii.gym.profile;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+@Configuration
+@Profile("dev")
+public class Dev {
+}
```

Comment:
Good to have this but the class is empty and does nothing. Is there another way to define profile? Please check for application.properties